### DISCUSSION TITLE
Step-by-Step Guide to Use OpenVINO in Proxmox Docker Container for Frigate

### DISCUSSION THREAD

**Discussion by crckrs85** [blakeblackshear/frigate#7128] [Ask A Question]
Hello everyone,

I'm currently working on a project where I want to use Frigate in a Proxmox environment, but I'm having some difficulty integrating OpenVINO into a Docker container on Proxmox.

Has anyone had experience setting this up and would be willing to share a step-by-step guide on how to do this?

Here's what I've done so far:

I've successfully installed Proxmox and am comfortable using it.
I've set up Docker on Proxmox.
I've run other containers successfully, but I'm having issues with the OpenVINO and Frigate setup.
Here are my specific questions:

How should I configure Docker to use OpenVINO in this environment?
Are there any specific settings or configurations I need to be aware of to run Frigate with OpenVINO on Docker?
If anyone has a sample Dockerfile or docker-compose file that's been used to set up a similar environment, could you share it?
Any help or pointers to relevant documentation would be highly appreciated. Thanks in advance!

**Comment by klingon888** [comment]
Here's a good step by step which worked for me - https://github.com/blakeblackshear/frigate/discussions/5773. Since you're using OpenVino, skip step 2 for Coral but pay particular attention to Step 3) Passing through the iGPU

Then follow the Openvino config as per https://docs.frigate.video/configuration/detectors

**Comment by pratbhoir** [comment]
Hi @crckrs85 
Any success through OpenVINO for frigate?

**Comment by jsapede** [comment]
here it is : 

https://github.com/jsapede/frigate-proxmox-docker-openvino
